libs/brig-types/stack.yaml

```diff
@@ -10,5 +10,6 @@ flags:
     protobuf: True
     arbitrary: True
 
-extra-deps: []
+extra-deps:
+- currency-codes-2.0.0.0
```

Comment:
why is this needed?

Reply:
No `currency-codes` on [stackage](https://www.stackage.org/package/currency-codes) unfortunately

Reply:
as far as I understand, this will end up in Stripe at some point? Would it be worth looking into `stripe-core`? it seems to enumerate the currencies supported by Stripe:
https://www.stackage.org/haddock/lts-10.2/stripe-core-2.2.3/Web-Stripe-InvoiceItem.html#t:Currency

Reply:
Stripe works slightly different. Customer is created independently from the currency. Then `ibis` creates new _Subscription_, for this new _Customer_, by choosing between various _Plans_ ( `monthly_eur`, `monthly_usd`, `monthly_chf`).  Those _Plans_ were previously created in Stripe Dashboard.

Reply:
@dkovacevic I think @elverkilde's point was just that instead of using [currency-codes](https://hackage.haskell.org/package/currency-codes) to manage currencies we could as well just use [stripe-core](https://www.stackage.org/haddock/lts-10.2/stripe-core-2.2.3/Web-Stripe-InvoiceItem.html#t:Currency) which has sort of the same functionality (with the added value that it already rejects some currencies that stripe will anyway not accept)